Utils/src/JetProperties.cc

```diff
@@ -127,6 +127,21 @@ DEFAULT_NAMED_PTR(D,neutralHadronPuppiMultiplicity);
 DEFAULT_NAMED_PTR(D,photonPuppiMultiplicity);
 DEFAULT_NAMED_PTR(D,msd); // softdrop mass as userfloat rather than from subjets
 
+class NamedPtr_pileupId : public NamedPtr<double> {
+	public:
+		using NamedPtr<double>::NamedPtr;
+		//default for user floats
+		void get_property(const pat::Jet& Jet) override { 
+                        if(Jet.pt() < 50.0 ){
+                                push_back(Jet.userFloat(extraInfo.at(0)));
+                        } else {
+                                push_back(-HUGE_VAL);
+                        }
+                }
+};
+
+DEFAULT_NAMED_PTR(pileupId,pileupJetId); 
```

Comment:
use `DEFINE_NAMED_PTR()` as is done for the other specialized helper classes below